ner_v2/detectors/numeral/number/number_detection.py

```diff
@@ -172,7 +172,11 @@ def detect_entity(self, text, **kwargs):
 
             for number_value_dict, original_text in zip(number_data[0], number_data[1]):
                 prev_original_text = prev_original_text + " " + original_text
-                number_value = int(str(number_value) + str(number_value_dict[NUMBER_DETECTION_RETURN_DICT_VALUE]))
+                try:
+                    number_value = float(
+                        str(number_value) + str(number_value_dict[NUMBER_DETECTION_RETURN_DICT_VALUE]))
+                except Exception:
+                    break
```

Comment:
Can you log in the case of exception and is breaking from the for loop intended?

Reply:
The exception is expected to be thrown every time concatenation strings are not compatible, and will be thrown on multiple occasions.
I am not sure whether we should be cluttering the logs with that.
Will add a comment explaining that inside the Exception. Let me know if you still want me to add the logline

Reply:
Got it, it's okay if we do not add any logging here then, and about the break, aren't we supposed to continue in the loop instead of the break?